### ISSUE TITLE
Support for R

### ISSUE THREAD

**Issue by haskellcamargo** [ramda/eslint-plugin-ramda#17]

**Comment by lo1tuma** [comment]
I’ve worked on this a while ago but haven’t had time to finish it, see 013b2a5bd20e60d7d81484be365ed00b10193ed4.

**Comment by lo1tuma** [comment]
> Currently, we support linting code imported with:
> ```import { ifElse, cond } from 'ramda'```
> and:
> ```import R from 'ramda'```

Currently we don’t even check imports, so e.g. the following code would cause false-positive problems reported by `eslint-plugin-ramda`:

```
import R from 'ramda';

function foo() {
  const R = { filter: 'foo' };
  R.filter(R.complement(R.T));
}
```

**Comment by 1024gs** [comment]
Would this enhancement cover the following case too?
```
import * as _ from aramda'
```

**Comment by haskellcamargo** [comment]
I think this should be covered.
Currently, there is no way to change the `R`, but I'll be working this weekend on, at least, a custom option to set the name of the Ramda object.
